src/SDK/DescopeSDK.php

```diff
@@ -127,14 +173,20 @@ public function logout(string $refreshToken): void
     }
 
     /**
-     * Logout a user from all devices.
+     * Logout a user from all devices using the refresh token.
      *
-     * @param  string $refreshToken The refresh token of the user.
+     * @param  string|null $refreshToken The refresh token of the user.
      * @return void
-     * @throws AuthException if the logout operation fails.
+     * @throws AuthException
      */
-    public function logoutAll(string $refreshToken): void
+    public function logoutAll(string $refreshToken = null): void
     {
+        $refreshToken = $refreshToken ?? $_COOKIE[EndpointsV1::REFRESH_COOKIE_NAME] ?? null;
+
+        if (!$refreshToken) {
+            throw new \InvalidArgumentException('Refresh token is required.');
```

Comment:
This shouldn't be. It was returning null or getting the `$refreshToken`. Now I'm just adding the ability to check the `COOKIE` value.

As for the check. I can change it to keep throwing the AuthException and remove that if statement, in case someone is developing around that specific Exception instead of InvalidArgument